README.md

```diff
@@ -251,8 +251,14 @@ A few ideas for where to take this in the future:
 ### Installation
 
 * `git clone` this repository
-* `npm install`
+* `yarn install`
+
+### Creating A New Rule
```

Comment:
Should be in a `CONTRIBUTING.md`, mind making an issue to track moving all of the "contributing related" items in the `README.md` into CONTRIBUTING.md?

Reply:
https://github.com/ember-template-lint/ember-template-lint/issues/1164